src/test/java/hudson/plugins/ec2/SlaveTemplateTest.java

```diff
@@ -217,7 +220,8 @@ public void testUnixConfigRoundTrip() throws Exception {
         AmazonEC2Cloud ac = new AmazonEC2Cloud("us-east-1", false, "abc", "us-east-1", "ghi", "3", templates, null, null);
         r.jenkins.clouds.add(ac);
 
-        r.submit(r.createWebClient().goTo("configureClouds").getFormByName("config"));
+        r.submit(getConfigForm(ac));
```

Comment:
Why submit twice?

Reply:
because i didn't carefully check my find and replace 🤦

Reply:
resolved in ca8172ba72f4cbaa7aadee92e5ad3f336e77034d